pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartBrokerCommand.java

```diff
@@ -67,6 +67,7 @@ public class StartBrokerCommand extends AbstractBaseAdminCommand implements Comm
       // TODO: support forbids = {"-brokerHost", "-brokerPort"})
   private String _configFileName;
 
+  @CommandLine.Option(names = {"-configOverrides"}, required = false, description = "Proxy config overrides")
```

Comment:
what's the intent for this commandline options?

Reply:
do we have a test for this? I am not sure picocli allows (Object) as the value type and does it automatically converts string into int/bool etc.

Reply:
I tried it locally and it works. 

I think it does allow. We could make it more type-bound by re-defining the map as a <string, string>. But I suspect that is what is happening by default. 
What I might be missing the multi-value regex-split parameter in the command line option. Let me add that.